### ISSUE TITLE
asm/enum `make` fails with no GOPATH

### ISSUE THREAD

**Issue by pwaller** [llir/llvm#82]
Further to #81:

```
:~/.local/src/github.com/llir/llvm/asm/enum$ make
make -C ../../tools
make[1]: Entering directory '/home/pwaller/.local/src/github.com/llir/llvm/tools'
go build github.com/mewspring/tools/cmd/string2enum
make[1]: Leaving directory '/home/pwaller/.local/src/github.com/llir/llvm/tools'
../../tools/string2enum -linecomment -type AtomicOp ../../ir/enum
stringer: unable to locate relative package import path for "../../ir/enum"; unable to locate path ("/home/pwaller/.local/src/github.com/llir/llvm/ir/enum") in GOPATH/src/ ("") or GOROOT/src/pkg/ ("")
github.com/mewkiz/pkg/goutil.RelImpPath
	/home/pwaller/go/pkg/mod/github.com/mewkiz/pkg@v0.0.0-20190317153131-e8616647c365/goutil/gotuil.go:62
main.main
	/home/pwaller/go/pkg/mod/github.com/mewspring/tools@v0.0.0-20181204020634-6c6637dc82b6/cmd/string2enum/stringer.go:143
runtime.main
	/home/pwaller/sdk/go1.12.1/src/runtime/proc.go:200
runtime.goexit
	/home/pwaller/sdk/go1.12.1/src/runtime/asm_amd64.s:1337
Makefile:11: recipe for target 'gen' failed
make: *** [gen] Error 1
```

**Comment by mewmew** [comment]
Ouch, yeah. We need to fix this.

**Comment by mewmew** [comment]
Resolved as of rev ec5154ad0c7b4fcfbd9834d1d0e700f9d8ce9594 and PR https://github.com/mewspring/tools/pull/1.
